docs/web3j-openapi.md

```diff
@@ -0,0 +1,248 @@
+# Web3j-OpenAPI
+
+[Web3j-OpenAPI](https://github.com/web3j/web3j-openapi) is an [OpenAPI](https://swagger.io/specification/) generator from [Solidity](https://solidity.readthedocs.io/) smart contracts. It provides a way to interact with the [Ethereum](https://ethereum.org/) blockchain via simple and intuitive HTTP requests. These interactions can be done using plain HTTP requests or via the [Swagger-UI](https://swagger.io/tools/swagger-ui/), which is generated with every project.
+
+This workflow can be summed in the following steps:
+- Writing a solidity smart contract
```

Comment:
Always capitalise `Solidity`

Reply:
Maybe mention what is the generation for:

> is an [OpenAPI](https://swagger.io/specification/) **client** and **server** generator from [Solidity](https://solidity.readthedocs.io/) smart contracts.

I also would first mention REST instead of HTTP the first time: 

> It provides a way to interact with [Ethereum](https://ethereum.org/) smart contracts using a simple RESTful API.